haiku-apps/projectconceptor/projectconceptor-0.1.1.recipe

```diff
@@ -0,0 +1,87 @@
+SUMMARY="A basic graph application"
+DESCRIPTION="ProjectConceptor is a modular application \
+designed to view, edit and process information which can be represented by a \
+graph structure. You can create nodes, name them and add attributes: \
```

Comment:
You may want to remove the "\" here to get a line-break in the description. Looks nicer than one lone paragraph.